docs/storage_options.md

```diff
@@ -6,6 +6,8 @@ Bandersnatch was originally developed for POSIX file system. Bandersnatch now su
 - [Amazon S3](https://aws.amazon.com/s3/)
 - [OpenStack Swift](https://docs.openstack.org/swift/latest/)
 
+(storage-backend-filesystem)=
+
```

Comment:
Is this work in progress?

Reply:
Aha, this is an explicit MyST ["explicit cross-reference target"](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#creating-explicit-targets) - `mdformat` then added a line between the target and the heading.

I added this when figuring out referencing across documents. I think I can remove it and use the auto-generated heading anchors instead. (MyST has what feels like dozens of ways to make cross references...)